ppr-ui/src/components/unitNotes/UnitNotePanels.vue

```diff
@@ -113,7 +122,11 @@ export default defineComponent({
 
     const localState = reactive({
       activePanels: [],
-      panelUnitNotes: createPanelUnitNotes(props.unitNotes)
+      panelUnitNotes: createPanelUnitNotes(props.unitNotes),
+      addUnitNoteDropdown: computed((): UnitNoteDocTypes[] => {
+        const dropdown = isRoleStaffReg ? ResidentialExemptionStaffDropDown : ResidentialExemptionQSDropDown
```

Comment:
@cameron-eyds @doug-lovett for awareness, I'm using this reg staff role (ppr_staff), but across the app we also do use 'staff'.

Reply:
Thanks Dima, the API looks at ppr_staff, not staff, when checking endpoint access.